src/components/v5/common/ActionsContent/partials/SocialLinksTable/partials/SocialLinksItem.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+import { useDetectClickOutside } from '~hooks';
+
+import useToggle from '~hooks/useToggle';
+import Icon from '~shared/Icon';
+import Card from '~v5/shared/Card';
+import styles from '../../TransactionItem/TransactionItem.module.css';
+import { SocialLinksItemProps } from '../types';
+
+const displayName =
+  'v5.common.ActionsContent.partials.SocialLinlsTable.partials.SocialLinksItem';
+
+const SocialLinksItem: FC<SocialLinksItemProps> = ({
+  type,
+  url,
+  id,
+  handleRemoveLink,
+  updateLink,
+}) => {
+  const { formatMessage } = useIntl();
+  const [
+    isTableMenuVisible,
+    { toggle: toggleTableMenu, toggleOff: toggleOffTableMenu },
+  ] = useToggle();
+  const ref = useDetectClickOutside({
+    onTriggered: () => toggleOffTableMenu(),
+  });
+
+  return (
+    <div className="flex items-center py-3 px-4 border-b border-gray-200 last:border-b-0 relative">
+      <div className="w-1/3 text-gray-900 text-1 capitalize">{type}</div>
+      <div className="w-2/3 text-gray-900 text-md">{url}</div>
+      <div className="absolute right-4 top-1/2 -translate-y-1/2">
```

Comment:
Why is this added using position absolute? Also, this looks like a kebab menu that is used everywhere across the app - shouldn't it be added as a separate component that you can reuse?